src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -698,10 +708,11 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
             }
           }
         }
-
+        // default group search with user input
         if ((this.type === PersonType.Group || this.type === PersonType.Any) && people.length < this.showMax) {
+          people = [];
           try {
-            const groups = (await findGroups(graph, input, this.showMax, this.groupType)) || [];
+            const groups = (await findGroups(graph, '', this.showMax, this.groupType)) || [];
```

Comment:
This is a bug, it should not be always searching for `''`

Reply:
fixed:
8836ca6

Reply:
Looks like you fixed it in the wrong place :(